File: specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/examples-js/StartManagedInstance.js
const { SqlManagementClient } = require("@azure/arm-sql");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Starts the managed instance.
 *
 * @summary Starts the managed instance.
 * x-ms-original-file: specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/examples/StartManagedInstance.json
 */
async function startsTheManagedInstance() {
  const subscriptionId =
    process.env["SQL_SUBSCRIPTION_ID"] || "00000000-1111-2222-3333-444444444444";
  const resourceGroupName = process.env["SQL_RESOURCE_GROUP"] || "startrg";
  const managedInstanceName = "mitostart";
  const credential = new DefaultAzureCredential();
  const client = new SqlManagementClient(credential, subscriptionId);
  const result = await client.managedInstances.beginStartAndWait(
    resourceGroupName,
    managedInstanceName,
  );
  console.log(result);
}

File: specification/sql/resource-manager/Microsoft.Sql/SQL/stable/2025-01-01/examples-java/StartManagedInstance.java

/**
 * Samples for ManagedInstances Start.
 */
public final class Main {
    /*
     * x-ms-original-file: 2025-01-01/StartManagedInstance.json
     */
    /**
     * Sample code: Starts the managed instance.
     * 
     * @param manager Entry point to SqlServerManager.
     */
    public static void startsTheManagedInstance(com.azure.resourcemanager.sql.SqlServerManager manager) {
        manager.serviceClient().getManagedInstances().start("startrg", "mitostart", com.azure.core.util.Context.NONE);
    }
}
